oap-server/server-starter/src/main/resources/ui-initialized-templates/rocketmq/rocketmq-topic.json

```
@@ -0,0 +1,299 @@
+          "w": 12,
+          "h": 15,
+          "i": "1",
+          "type": "Widget",
+          "id": "1",
+          "metricTypes": [],
+          "metrics": [],
+          "metricMode": "Expression",
+          "moved": false,
+          "graph": {
+            "type": "Bar",
+            "showBackground": true
+          },
+          "expressions": [
+            "meter_rocketmq_topic_consumer_message_size/1024/1024"
+          ],
+          "typesOfMQE": [
+            "TIME_SERIES_VALUES"
+          ],
+          "widget": {
+            "name": "consumer_message_size",
+            "title": "Consumer Message Size(MB)",
+            "tips": "The size of messages consumed per second "
+          }
+        },
+        {
+          "x": 6,
+          "y": 0,
+          "w": 6,
+          "h": 10,
+          "i": "2",
+          "type": "Widget",
+          "id": "2",
+          "metricTypes": [],
+          "metrics": [],
+          "metricMode": "Expression",
+          "metricConfig": [
+            {
+              "unit": "MB"
+            }
+          ],
+          "moved": false,
+          "expressions": [
+            "latest(meter_rocketmq_topic_max_consumer_message_size)/1024/1024"
+          ],
+          "typesOfMQE": [
+            "SINGLE_VALUE"
+          ],
+          "widget": {
+            "name": "max_consumer_message_size",
+            "title": "Max Consumer Message Size",
+            "tips": "The maximum number of messages consumed"
+          },
+          "graph": {
+            "type": "Card",
+            "fontSize": 40,
+            "textAlign": "center",
+            "showUnit": true
+          }
+        },
+        {
+          "x": 0,
+          "y": 0,
+          "w": 6,
+          "h": 10,
+          "i": "3",
+          "type": "Widget",
+          "id": "3",
+          "metricTypes": [],
+          "metrics": [],
+          "metricConfig": [
+            {
+              "unit": "MB"
+            }
+          ],
+          "metricMode": "Expression",
+          "moved": false,
+          "expressions": [
+            "latest(meter_rocketmq_topic_max_producer_message_size)/1024/1024"
+          ],
+          "typesOfMQE": [
+            "SINGLE_VALUE"
+          ],
+          "widget": {
+            "name": "max_producer_message_size",
+            "title": "Max Producer Message Size",
+            "tips": "The maximum number of messages produced"
+          },
+          "graph": {
+            "type": "Card",
+            "fontSize": 40,
+            "textAlign": "center",
+            "showUnit": true
+          }
+        },
+        {
+          "x": 12,
+          "y": 0,
+          "w": 12,
+          "h": 10,
+          "i": "5",
+          "type": "Widget",
+          "id": "5",
+          "metricTypes": [],
+          "metrics": [],
+          "metricMode": "Expression",
+          "moved": false,
+          "expressions": [
+            "avg(meter_rocketmq_topic_consumer_latency)/1000"
```

```suggestion
            "meter_rocketmq_topic_consumer_latency/1000"
```
the expression needs to be restored too.